src/ContainerHealthLogs.jsx

```diff
@@ -83,22 +76,22 @@ const ContainerHealthLogs = ({ container, containerDetail, onAddNotification, st
                             <DescriptionListTerm>{_("Timeout")}</DescriptionListTerm>
                             <DescriptionListDescription>{format_nanoseconds(healthCheck.Timeout)}</DescriptionListDescription>
                         </DescriptionListGroup>}
-                        {healthCheck.HealthcheckOnFailureAction && <DescriptionListGroup>
+                        {container.Config?.HealthcheckOnFailureAction && <DescriptionListGroup>
                             <DescriptionListTerm>{_("When unhealthy")}</DescriptionListTerm>
-                            <DescriptionListDescription>{HealthcheckOnFailureActionText[healthCheck.HealthcheckOnFailureAction]}</DescriptionListDescription>
+                            <DescriptionListDescription>{HealthcheckOnFailureActionText[container.Config.HealthcheckOnFailureAction]}</DescriptionListDescription>
                         </DescriptionListGroup>}
-                        {failingStreak !== 0 && <DescriptionListGroup>
+                        {healthState.FailingStreak && <DescriptionListGroup>
                             <DescriptionListTerm>{_("Failing streak")}</DescriptionListTerm>
-                            <DescriptionListDescription>{failingStreak}</DescriptionListDescription>
+                            <DescriptionListDescription>{healthState.FailingStreak}</DescriptionListDescription>
                         </DescriptionListGroup>}
                     </DescriptionList>
                 </FlexItem>
-                { container.State === "running" &&
+                { container.State.Status === "running" &&
                     <FlexItem>
                         <Button variant="secondary" onClick={() => {
                             client.runHealthcheck(container.isSystem, container.Id)
                                     .catch(ex => {
-                                        const error = cockpit.format(_("Failed to run health check on container $0"), container.Names);
+                                        const error = cockpit.format(_("Failed to run health check on container $0"), container.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)